webapp/src/pages/applicationForm/components/ApplicationForm.js

```diff
@@ -1138,34 +1406,104 @@ class ApplicationForm extends Component {
-      return <div>
-          <ApplicationList responses={responses} event={this.state.event} formSpec={formSpec} click={this.responseSelected} /><br />
-          <button className="btn btn-primary" onClick={() => this.newNomination()}>{newForm} &gt;</button>
-    </div>
+      return (
+        <div className={"alert alert-danger alert-container"}>
+          {errorMessage}
+        </div>
+      );
     }
-    else {
-      return <ApplicationFormInstance
-        formSpec={formSpec}
-        response={selectedResponse}
-        event={this.props.event} />
+
+    if (this.props.event.event_type === "JOURNAL") {
+      if (this.props.journalSubmissionFlag) {
+        return (
+          <ApplicationFormInstance
+            formSpec={formSpec}
+            response={selectedResponse.is_submitted ? null : selectedResponse}
+            event={this.props.event}
+          />
+        );
+      }
+      if (this.props.view) {
+        
+        return (
+          <div>
+            <ApplicationList
+              responses={responses}
+              event={this.state.event}
+              formSpec={formSpec}
+              click={this.responseSelected}
+            />
+            <br />
+            <button
+              className="btn btn-primary"
+              onClick={() => this.newSubmission()}
+            >
+              {this.props.t("New Submission")} &gt;
+            </button>
+          </div>
+        );
+      }
+      if (this.props.continue) {
+        return (
+          <ApplicationFormInstance
+            formSpec={formSpec}
+            response={listselectedResponse.find(
+              (item) => item.id === parseInt(this.props.match.params.id)
+            )}
+            event={this.props.event}
+          />
+        );
+      }
+      return (
+        <ApplicationFormInstance
+          formSpec={formSpec}
+          response={null}
+          event={this.props.event}
+        />
+      );
+    } else if (
+      formSpec.nominations &&
+      responses.length > 0 &&
+      !responseSelected
+    ) {
+      let newForm =
+        this.state.event.event_type === "JOURNAL"
+          ? this.props.t("New Submission") + " "
+          : this.props.t("New Nomination") + " ";
+      return (
+        <div>
+          <ApplicationList
+            responses={responses}
+            event={this.state.event}
+            formSpec={formSpec}
+            click={this.responseSelected}
+          />
+          <br />
+          <button
+            className="btn btn-primary"
+            onClick={() => this.newNomination()}
+          >
+            {newForm} &gt;
+          </button>
+        </div>
+      );
+    } else {
+      return (
+        <ApplicationFormInstance
+          formSpec={formSpec}
+          response={selectedResponse}
+          event={this.props.event}
+        />
+      );
     }
```

Comment:
In future, please can you keep formatting changes in a separate PR that only does formatting. When formatting changes are mixed up with other logic changes it makes it very hard for a reviewer to see what has actually changed